packages/extensions/src/table/table-utils/get-cells-in-row.ts

```diff
@@ -0,0 +1,40 @@
+import type { Selection } from '@prosekit/pm/state'
+import { TableMap } from 'prosemirror-tables'
+
+import { findTable } from './query'
+import type { CellPos } from './types'
+
+/**
+ * Returns an array of cells in a row(s), where `rowIndex` could be a row index or an array of row indexes.
+ *
+ * @internal
+ */
+export function getCellsInRow(rowIndex: number | number[]) {
+  return (selection: Selection): CellPos[] | undefined => {
```

Comment:
```suggestion
export function getCellsInRow(rowIndex: number | number[], selection: Selection): CellPos[] | undefined {
```

I don't think we need a function return another function. We're not caching the returned function. 


*Yes I know this function is taken from Atlassian. But they're not caching the returning function either, so I don't see a reason to design an API like this*


<img width="930" alt="image" src="https://github.com/user-attachments/assets/f03d2042-5105-4803-88f3-408f1546711b" />

Reply:
Fixed by https://github.com/prosekit/prosekit/pull/1109/commits/467d161a2f286fff50493bcf8b0834f5ae07c0d1